prov/util/src/util_attr.c

```diff
@@ -220,6 +220,7 @@ int ofix_getinfo(uint32_t version, const char *node, const char *service,
 					fi, alter_base_info, &temp);
 			if (ret)
 				goto err3;
+			ofi_alter_info(temp, hints);
```

Comment:
How important is this change?  I'm asking relative to 1.4.0rc2, which I'm in the middle of preparing.